extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -379,6 +382,10 @@ class AmpAd extends AMP.BaseElement {
    * @private
    */
   noContentHandler_() {
+    // If iframe is null nothing to do.
```

Comment:
Oh, right. The same checks needs to be done at the start of the `deferMutate` callback.